examples/notebooks/load-modelling.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load Shedding, Shifting and Elastic Demand"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook provides a simple illustration of how load shedding, load shifting, and elastic demand can be modeled in PyPSA. Through the example, we'll simulate a single day of electrical consumption in a network with solar generation, which could represent a household or a small community.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pypsa\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Build example network\n",
+    "\n",
+    "We'll start by constructing a minimal example network to demonstrate the demand modeling cases. The network consists of:\n",
+    "\n",
+    "- a single bus with a time span between 04:00 - 20:00.\n",
```

Comment:
Not sure I get the point here. Does it mean that we consider only 04:00-20:00? It  would be also great to specify what does the bus represent, assuming that a reader can be not perfectly fluent with PyPSA components. So, a short clarification could be very helpful, like: "a bus which represents a network node, to which generation and loads are connected".

Reply:
Edited to:
"## Build example network

We'll start by constructing a minimal example network to demonstrate the demand modeling cases. The network consists of:

- a single bus which represents a network node, to which generation and loads are connected.
- an hourly time span of a single day, from 00:00 to 23:00.
- one PV (Photovoltaic) generator, generating only in the daytime.
- an electricity load profile (active), with peak load between 19:00 - 23:00." ✔️

Reply:
Sounds great! 👍🏽